app/jobs/import_distribution_points_job.rb

```diff
@@ -5,11 +5,9 @@ def perform(*args)
     logger.info "Starting ImportDistributionPointsJob #{Time.now}"
     distribution_points = APIImporter.distribution_points
     distribution_points.each do |distribution_point|
-      # updated_at, last_updated are derived fields
+      # last_updated are derived fields
       # updatedAt is set on save to the database
-      DistributionPoint.create! distribution_point.except(
-        'updated_at', 'updatedAt', 'last_updated'
-      )
+      DistributionPoint.create! distribution_point.except('updatedAt', 'last_updated')
```

Comment:
Only derived fields not present in database columns were being stripped elsewhere, but it's moot either way, I just checked my local database and updated_at gets overwritten.  I can think of a situation where this could be problematic if someone is trying to reflect production data to solve an issue with dependent on reviewing outdated records (especially with the `Make One Call` feature).

Reply:
OK, in that case, good to merge.